### PR TITLE
Finish documentation in include/3ds and include/3ds/util.

### PR DESCRIPTION
**Pull request by Steveice10**
This PR finishes up the documentation in include/3ds and include/3ds/util. Some GPU, NDSP, and service headers remain.

### CODE REVIEW

File: libctru/include/3ds/srv.h [hunk 1]
```diff
@@ -1,14 +1,73 @@
+/**
+ * @file srv.h
+ * @brief Service API.
+ */
 #pragma once
 
+/**
+ * Initializes the service API.
```
Comment (Lectem) [inline libctru/include/3ds/srv.h]: `@brief`

File: libctru/include/3ds/srv.h [hunk 2]
```diff
@@ -1,14 +1,73 @@
...
+ */
 Result srvInit(void);
+
+/**
+ * Exits the service API.
```
Comment (Lectem) [inline libctru/include/3ds/srv.h]: `@brief`

**Comment by Steveice10** [comment]
Missing brief tags should be fixed.

**Comment by Lectem** [comment]
LGTM

**Pull request merged**